models/org_test.go

```diff
@@ -240,7 +240,8 @@ func TestCountOrganizations(t *testing.T) {
 func TestOrganizations(t *testing.T) {
```

Comment:
This test should be moved from `org_test` to `user_test`

Reply:
This test will only set `type = UserTypeOrganization`, it's for organizations but not users. Since `Users` and `Organizations` share the `user` table. `SearchUsers` will be shared also.

Reply:
But it tests a function that is defined in `user.go`, whereas it previously tested a function that was defined in `org.go`. It is my understanding that if something is defined in `XYZ.go`, its test should be in `XYZ_test.go`; is my understanding incorrect?

Reply:
But should I rename it from `TestOrganizations` to `TestUsers` or just put `TestOrganizations` to `user_test.go`? This function logically test `organizations` but the function is shared by `Users` and `Organizations`. So I don't think we can obey the rule you mentioned above.

Reply:
I would suggest renaming it to `TestSearchUsers`, and expanding it to test for both individual users and organizations, but you can do what you think is best.

One way or another, however, we should make sure that the `SearchUsers` function is fully tested (both individual users and organizations).